packages/truffle/test/scenarios/migrations/migrate.js

```diff
@@ -46,7 +46,6 @@ describe("migrate (success)", function() {
     assert(output.includes("Replacing 'UsesExample'"));
     assert(output.includes("PayableExample"));
     assert(output.includes("1 ETH"));
-    assert(output.includes("Saving migration"));
```

Comment:
Wait but what happened to this output?

Reply:
It was shown as an ora spinner, which conflicted with the one that we needed to show.

Also it wasn't true "output," as it was sent to stderr and deleted on the `endTransaction` event (see edits to `Reporter.js` and `Message.js` in `events`)

Reply:
Ah, so we're just not doing that behavior anymore. Seems fine, but might want a second opinion from @eggplantzzz

Reply:
Another option would be to retain this, but strip out `ora` and use a lib like [`spinnies`](https://github.com/jcarpanelli/spinnies) in its place.

Reply:
Has been updated to use `spinnies` now (see `packages/truffle/spinners`)

Reply:
What are the tradeoffs ora vs. spinnies? I'm not in love with ora, is spinnies a better package you think?

Reply:
spinner work has been moved to #4934